ui-react/packages/atlasmap-core/src/Atlasmap.tsx

```diff
@@ -136,11 +137,14 @@ export const Atlasmap: FunctionComponent<IAtlasmapProps> = ({
               sources={sources}
             />
 
-            <AtlasmapCanvasViewMappings onAddToMapping={addToMapping} />
+            <AtlasmapCanvasViewMappings
+              onAddToMapping={addToMapping}
+              onCreateMapping={createMapping}
```

Comment:
You’ll have to remove this too I think

Reply:
*ugh* - thought I needed that one - still new at this...